app/services/players_service.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+from uuid import UUID
+
+from app.core.config import settings
+
+from .base_service import BaseService
+
+
+class PlayersService(BaseService):
+    def __init__(self) -> None:
+        """Init the service."""
+        super().__init__()
+        self.name = "players-service"
+        self._set_base_url(
+            True, settings.PLAYERS_SERVICE_HOST, settings.PLAYERS_SERVICE_PORT
+        )
+        if settings.PLAYERS_SERVICE_API_KEY:
+            self.set_base_headers({"x-api-key": settings.PLAYERS_SERVICE_API_KEY})
```

Comment:
¿En que momento no sería necesaría la API_KEY del player?
Sugerimos que sea obligatorio
```suggestion
        self.set_base_headers({"x-api-key": settings.PLAYERS_SERVICE_API_KEY})
```